cookbook/configuration/mongodb_session_storage.rst

```diff
@@ -0,0 +1,54 @@
+How to Use MongoDbSessionHandler to Store Sessions in a MongoDB Database
+========================================================================
+
+The default Symfony session storage writes the session information to files.
+Some medium to large websites use a NoSQL database called MongoDB to store the
+session values instead of files, because databases are easier to use and scale
+in a multi-webserver environment.
+
+Symfony has a built-in solution for NoSQL database session storage called
+:class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MongoDbSessionHandler`.
+MongoDB is an open-source document database that provides high performance,
+high availability, and automatic scaling. This article assumes that you have
+already `installed and configured a MongoDB server`_. To use it, you just
+need to change/add some parameters in the main configuration file:
+
+.. configuration-block::
+
+    .. code-block:: yaml
```

Comment:
I'm not sure if we should configure the parameters in all code examples. Usually one would simply put them in their `parameters.yml` file and just refers to them in the service configuration. Imho we should simply show the service configurations and add an explanation that these parameters need to be defined somewhere (like you do with other database connections as well).